packages/mobile/src/account/BankAccounts.tsx

```diff
@@ -0,0 +1,241 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useLayoutEffect, useState } from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View, Image } from 'react-native'
+import { useSelector } from 'react-redux'
+import PlusIcon from 'src/icons/PlusIcon'
+import TrippleDotVertical from 'src/icons/TrippleDotVertical'
+import { deleteFinclusiveBankAccount, getFinclusiveBankAccount } from 'src/in-house-liquidity'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import Logger from 'src/utils/Logger'
+import { dataEncryptionKeySelector, mtwAddressSelector } from 'src/web3/selectors'
+import BorderlessButton from '@celo/react-components/components/BorderlessButton'
+import { navigate } from 'src/navigator/NavigationService'
+import openPlaid from './openPlaid'
+import { plaidParamsSelector } from 'src/account/selectors'
+import OptionsChooser from 'src/components/OptionsChooser'
+
+const TAG = 'BankAccounts'
+
+type Props = StackScreenProps<StackParamList, Screens.BankAccounts>
+function BankAccounts({ navigation, route }: Props) {
+  const { t } = useTranslation()
+  const [isOptionsVisible, setIsOptionsVisible] = useState(false)
+  const [selectedBankId, setSelectedBankId] = useState(0)
+  const accountMTWAddress = useSelector(mtwAddressSelector) || ''
+  const dekPrivate = useSelector(dataEncryptionKeySelector)
+  const plaidParams = useSelector(plaidParamsSelector)
+  const { newPublicToken } = route.params
+
+  const header = () => {
+    return (
+      <View style={styles.header}>
+        <Text style={fontStyles.navigationHeader}>{t('bankAccountsScreen.header')}</Text>
+      </View>
+    )
+  }
+
+  useLayoutEffect(() => {
+    navigation.setOptions({
+      headerTitle: header,
+    })
+  }, [navigation])
+
+  const bankAccounts = useAsync(async () => {
+    if (!dekPrivate) {
```

Comment:
should there be a similar check for when the MTW address is null (or in this case, an empty string)?

Reply:
if it's null/empty, seems like it isn't useful to query IHL

Reply:
added the check